notebook/base/handlers.py

```diff
@@ -849,7 +850,7 @@ class APIVersionHandler(APIHandler):
 
     def get(self):
         # not authenticated, so give as few info as possible
-        self.finish(json.dumps({"version":notebook.__version__}))
+        self.finish(json.dumps({"version": notebook.__version__, "module": JUPYTER_NOTEBOOK_TAG}))
```

Comment:
I don't think we can do this as it will break clients.  This is something that would need to be reserved for a major release.

Instead of hitting `/api`, we should hit either `/api/status` - which returns something like the following:
```
{"connections": 0, "kernels": 0, "last_activity": "2020-09-03T14:11:03.989420Z", "started": "2020-09-03T14:10:45.100779Z"}
```
or `/api/spec.yaml` to get the swagger/OpenAPI doc, the first portion of which looks as follows:
```
swagger: '2.0'
info:
  title: Jupyter Notebook API
  description: Notebook API
  version: "5"
  contact:
    name: Jupyter Project
    url: https://jupyter.org
```
I think interpreting either of these results would be a definitive approach that spans versions.

Seems like hitting `/api/status` and confirming "connections" and "kernels" fields are present is probably the most appropriate for this exercise.

Reply:
clients feel good; you see all tests passed.
also i feel good you can ask so comprehensive consideration.